docs/source/installation.rst

```diff
@@ -1,10 +1,63 @@
+.. _installation:
+
 Installation
 ============
 
-TODO
+Installing from PyPi
+-------------------------
+
+ETNA is available on `PyPI <https://pypi.org/project/etna>`_, so you can use ``pip`` to install it.
+
+Install default version:
+
+.. code-block:: console
+
+    pip install --upgrade pip
+    pip install etna
+
+The default version doesn't contain all the dependencies, because some of them are needed only for specific models, e.g. Prophet, PyTorch.
+Available user extensions are the following:
+* ``prophet``: adds prophet model`,
+* ``torch``: adds models based on neural nets,
+* ``wandb``: adds wandb logger,
+* ``auto``: adds AutoML functionality,
+* ``statsforecast``: adds models from `statsforecast <https://nixtla.github.io/statsforecast/>`_,
+* ``classiciation``: adds time series classification functionality.
+
+Install extension:
+
+.. code-block:: console
+
+    pip install etna[extension-name]
```

Comment:
On some platforms you need to write pip install 'etna[extension-name]', we can also mention it

Reply:
Ok, I forgot to enclose it in quotes, will add it.